terraform/workspace-variables/app_config.yml

```diff
@@ -57,7 +57,7 @@ content:
   # DASHBOARD_UPDATE_INTERVAL: '*/5 * * * *' # every 5 mins (default: daily at noon)
   # cms
   # CONTENTFUL_PREVIEW: true
-  CONTENTFUL: false
+  CONTENTFUL: true
```

Comment:
This is the default so you can remove the line and not need to commit any change to the file.

Reply:
OK [26052aa](https://github.com/DFE-Digital/early-years-foundation-recovery/pull/659/commits/26052aac0051d052d39fd017f178cf1ad1d8a5c7)